static/js/composebox_typeahead.js

```diff
@@ -343,13 +343,15 @@ export function tokenize_compose_str(s) {
 }
 
 export function broadcast_mentions() {
+    const wildcard_mention_array = ["all", "everyone"];
```

Comment:
Since this doesn't contain the sorting part, I think you should make the commit message say "Fixes part of #21643"; otherwise merging this will automatically close that issue :)